### PR TITLE
ENG-7873 location service fix and ENG-7840 add adv libs to release

### PR DESCRIPTION
**Pull request by kishisaka-nid**
This includes the location service fix and adding advanced libs to the 3.2.2 release that should happen tomorrow.

### CODE REVIEW

**Review by priya-xavier — APPROVED** [review]
LGTM!

**Review by priya-xavier — APPROVED** [review]
LGTM!

File: version.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #Fri May 27 05:08:14 UTC 2022
-SDK_VERSION=3.2.1
+SDK_VERSION=3.2.2
 VERSION_MAJOR=3
 VERSION_MINOR=2
-VERSION_PATCH=1
+VERSION_PATCH=2
```
Comment (kevinsites) [inline version.properties:5]: Do we want to include this right now or should we wait till we cut a release from this branch?

Comment (kishisaka-nid) [reply version.properties:5]: this PR merge to the release branch will trigger the release. we won’t be manually running the update-version workflow

**Review by kevinsites — APPROVED** [review]

**Pull request merged**